src/other/Table.js

```diff
@@ -63,13 +63,48 @@
             .remove()
         ;
 
-        if (this.pagination()) {
+        var theadHeight = parseInt(th.style('height').replace('px',''));
+
+        // Create temp row and calculate its height
+        var trow = this.tbody.selectAll("tr").data([[0]]);
+        trow
+            .enter()
+            .append("tr")
+        ;
+        var tcell = trow.selectAll("td").data(function (row, i) {
+            return row;
+        });
+        tcell.enter()
+            .append("td")
+        ;
+		tcell
+            .text(function (d) {
+                return d;
+            })
+        ;
+
+        tcell.exit()
+            .remove()
+        ;
+
+        var tcellHeight = parseInt(th.style('height').replace('px',''));
```

Comment:
th should be tcell ... (this prolly explains why the padding wasnt adding up to what i thought)